test/index_tests.js

```diff
@@ -112,6 +114,19 @@ describe('index.js', () => {
     });
   });
 
+  it('WHEN tags are supplied THEN path should contain the matching ones', done => {
+    const tag = 'hello';
```

Comment:
Once again, here perhaps it's not that important, but even for maintaining, try sticking to something that resembles a possible use case, simply for a better mental model.

In this case, `user` or `users` probably:D

Plus, it'll be more consistent, and that's almost always better:D

Reply:
I completely agree! I just stick to the existing tests. I'll leave the existing dataset as it was and create a new one for this case.

Reply:
Created a new test with `Students` tag.